src/Game.cpp

```diff
@@ -386,6 +384,37 @@ bool Game::Update()
 
 bool Game::Run()
 {
+	Locator::resources::set<resources::Resources>();
+	auto& resources = Locator::resources::ref();
+	auto& meshManager = resources.GetMeshes();
+	auto& textureManager = resources.GetTextures();
+	auto& animationManager = resources.GetAnimations();
+
+	meshManager.Load("hand", "hand", (_fileSystem->CreatureMeshPath() / "Hand_Boned_Base2.l3d").u8string());
+	meshManager.Load("testModel", "test_model", (_fileSystem->MiscPath() / "coffre.l3d").u8string());
+	pack::PackFile pack;
+	pack.Open(Game::instance()->GetFileSystem().FindPath(_fileSystem->DataPath() / "AllMeshes.g3d").u8string());
+	auto& meshes = pack.GetMeshes();
+	for (auto i = 0; i < meshes.size(); i++)
```

Comment:
Getting some warnings here about the type of `i = 0` being int instead of size_t